src/app/store/models/IPlayers.state.ts

```diff
@@ -0,0 +1,13 @@
+export interface IPlayer {
```

Comment:
The Player need status or readystate?

Reply:
Good point, I'll add it, but remember that it's provisional since the interface agreement isn't there yet. When you have it, the Store interfaces will be updated